src/NServiceBus.Msmq.AcceptanceTests/When_no_explicit_error_queue_is_configured.cs

```diff
@@ -0,0 +1,128 @@
+﻿namespace NServiceBus.AcceptanceTests.Recoverability
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Messaging;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using AcceptanceTesting.Customization;
+    using AcceptanceTesting.Support;
+    using Config.ConfigurationSource;
+    using Configuration.AdvanceExtensibility;
+    using EndpointTemplates;
+    using NUnit.Framework;
+    using Transport;
+
+    public class When_no_explicit_error_queue_is_configured : NServiceBusAcceptanceTest
+    {
+        [Test]
+        public void Should_not_start_endpoint()
+        {
+            var ex = Assert.ThrowsAsync<Exception>(async () =>
+            {
+                await Scenario.Define<Context>()
+                    .WithEndpoint<EndpointWithNoErrorQConfig>()
+                    .Run();
+            });
+
+            StringAssert.Contains("Faults forwarding requires an error queue to be specified", ex.Message);
+        }
+
+        public class EndpointWithNoErrorQConfig : EndpointConfigurationBuilder
+        {
+            public EndpointWithNoErrorQConfig()
+            {
+                EndpointSetup<PlainVanillaEndpoint>();
+            }
+        }
+
+        public class Context : ScenarioContext
+        {
+        }
+
+        public class PlainVanillaEndpoint : IEndpointSetupTemplate
```

Comment:
theoretically we could make a base class `DefaultServer`  inherits from, which does not specify the error queue, but this would be kinda weird too. :(

We could save us the cleanup mechanism as the endpoint should never start in the first place?